adal/src/main/java/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -615,7 +602,19 @@ public void processRedirectUrl(final WebView view, String url) {
                 // access token
                 new TokenTask(mWebRequestHandler, mAuthRequest, mCallingPackage, mCallingUID)
                         .execute(url);
+                return;
```

Comment:
ok, will fix and lets discuss.

---
In reply to: [73024085](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/689#discussion_r73024085) [](ancestors = 73024085)